modules/vacunas/schemas/vacunasApi.ts

```diff
@@ -13,10 +14,37 @@ export let schema = new mongoose.Schema({
     },
     vacuna: String,
     dosis: String,
+    lote: String,
     fechaAplicacion: Date,
     efector: String,
     esquema: String,
-    condicion: String
+    condicion: String,
+    paciente: {
+        id: Types.ObjectId,
+        zona: DireccionSchema,
+        contacto: ContactoSchema
+    },
```

Comment:
@negro89 consulta: en el schema ya existen datos demográficos (nombre, apellido, dni, etc) presumiblemente del paciente. Esos datos no deberían estar contenidos dentro del property `paciente`?

En caso de que el paciente de vacunaApi sea una version acotada del schema de Paciente, no debería zona llamarse `direccion` y ser un array de DireccionSchema (tal como en el schema de Paciente)?
```suggestion
    paciente: {
        id: Types.ObjectId,
        direccion: [DireccionSchema],
        contacto: ContactoSchema
    },
```

Reply:
En verdad se uso la colección incorrecta. Esa colección es de nomivac. Lo que se pretendía hacer iba en una colección nueva. 
De todas forma, va a cambiar un poco este modelo.